lib/screen/cloud.dart

```diff
@@ -0,0 +1,255 @@
+                  boxShadow: [
+                    BoxShadow(
+                      color: Colors.black,
+                      offset: const Offset(
+                        1.0,
+                        1.0,
+                      ), //Offset
+                      blurRadius: 30.0,
+                      spreadRadius: 1.0,
+                    ),
+                  ],
+                ),
+                padding: EdgeInsets.only(top: 30.0),
+                height: screensize.height * 0.35,
+                child: Column(
+                  children: [
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                      children: [
+                        IconButton(
+                          onPressed: null,
+                          icon: ImageIcon(
+                            AssetImage(Back),
+                            size: 50.0,
+                            color: Colors.white,
+                          ),
+                        ),
+                        Container(
+                          alignment: Alignment.centerLeft,
+                          child: Padding(
+                            padding: EdgeInsets.symmetric(horizontal: 15.0),
+                            child: Text(
+                              "Cloud",
+                              textAlign: TextAlign.left,
+                              style: GoogleFonts.raleway(
+                                textStyle: TextStyle(
+                                  fontSize: 35,
+                                  fontWeight: FontWeight.w600,
+                                  color: Colors.white,
+                                  // letterSpacing: 1.2,
+                                ),
+                              ),
+                            ),
+                          ),
+                        ),
+                        RotatedBox(
+                          quarterTurns: 2,
+                          child: IconButton(
+                            onPressed: null,
+                            icon: ImageIcon(
+                              AssetImage(Back),
+                              size: 50.0,
+                              color: Colors.white,
+                            ),
+                          ),
+                        ),
+                      ],
+                    ),
+                    Flexible(
+                      child: Padding(
+                        padding: const EdgeInsets.only(top: 10.0),
+                        child: CloudCarousel(),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ],
+          ),
+          Container(
+            margin: const EdgeInsets.only(top: 10.0),
+            height: screensize.height * 0.635,
+            child: ListView(
+              shrinkWrap: true,
+              scrollDirection: Axis.vertical,
+              children: [
```

Comment:
All the children of this ListView should also be generated the same way as cloud_carousel.dart